### ISSUE TITLE
[feature] link upload and/or additional buttons with custom functions

### ISSUE THREAD

**Issue by evesdropper** [gabm/Satty#89]
Wondering if it'd be possible to have a button that uploads to an image server (e.g. I currently have a script that uploads to imgbb) and copies the link onto clipboard like how lightshot (which I used on Windows in the past) worked.

This might force people to use one specific image server so I was thinking if not then maybe it could be possible to put additional buttons/keymaps that could map to custom functions such as uploading to an image server.

**Comment by gabm** [comment]
I am not against it. But I am not sure, if we have the time to develop and maintain those features?

**Comment by sand4rt** [comment]
Yeah It would be great if there was an option to add custom keybindings. In swayimg i have a "Mail image" _(opens a mail client with the image attached)_ and "Ripdrag" _(drag‑and‑drop)_ keybindings that i'd like to use in satty.
